client/cli/go/cmds/volume.go

```diff
@@ -475,7 +475,7 @@ Block Hosting Restriction: {{.BlockInfo.Restriction}}
 Block Volumes: {{.BlockInfo.BlockVolumes}}
 Durability Type: {{.Durability.Type}}
 {{- if eq .Durability.Type "replicate" }}
-Distributed+Replica: {{.Durability.Replicate.Replica}}
+Distributed+Replica: {{ . | numBrickSets }}x{{.Durability.Replicate.Replica}}
```

Comment:
I think we should not print `Distributed+Replica` here. Besides being somewhat grammatically questionable, I think it is better and clearer to separate `Distribute` and `Replica` counts into two lines. The Distribute count is also applicable to other durability types and should be printed for them as well.